src/config/config_test.go

```diff
@@ -13,6 +13,61 @@ func TestConfig(t *testing.T) {
 	RunSpecs(t, "config_test")
 }
 
+var _ = Describe("ProcessArgs", func() {
+
+	It("Should not deserialize installer args when they are not supplied.", func() {
```

Comment:
This seems like a duplicate of the test in `SetInstallerArgs`.
I think you'd be safe making that method private and removing those tests I don't expect that method to be useful outside of the package.

Same for all the other duplicated tests here.